docs/schema.json

```diff
@@ -3414,25 +3434,30 @@
         },
         "finalPriceConsent": {
           "typeRef": "#/domains/Generic/types/PriceConsent",
-          "description": "Client's consent for final price, should exactly match the <code>finalPrice</code> object from output.<br/>Automation will not proceed with placing order until the consent is provided."
+          "description": "Client's consent for final price, should exactly match the <code>finalPrice</code> object from output.<br/>Automation will not proceed with placing order until the consent is provided.",
+          "fooBarBaz": "finalPrice"
         },
         "selectedDelivery": {
           "typeRef": "#/domains/EventBooking/types/Delivery",
-          "description": "Requested when delivery choices are required by website.<br/>At this point <code>availableDeliveries</code> output should contain information about available delivery options.<br/>Note: on deep links with pre-selected delivery options, this input might not required.<br/>Automation may fail if the incorrect option is specified, or if the delivery option is no longer available, for example, for a late or same day booking."
+          "description": "Requested when delivery choices are required by website.<br/>At this point <code>availableDeliveries</code> output should contain information about available delivery options.<br/>Note: on deep links with pre-selected delivery options, this input might not required.<br/>Automation may fail if the incorrect option is specified, or if the delivery option is no longer available, for example, for a late or same day booking.",
+          "fooBarBaz": "availableDeliveries"
         },
         "options": {
           "typeRef": "#/domains/EventBooking/types/Options",
           "initial": true
         },
         "selectedRefund": {
-          "typeRef": "#/domains/EventBooking/types/SelectedRefund"
+          "typeRef": "#/domains/EventBooking/types/SelectedRefund",
+          "fooBarBaz": "availableRefunds"
         },
         "selectedSeats": {
-          "typeRef": "#/domains/EventBooking/types/SelectedSeats"
+          "typeRef": "#/domains/EventBooking/types/SelectedSeats",
+          "fooBarBaz": ""
```

Comment:
this is wrong, `selected` prefix was confusing, actually this input doesn't depend on any output